### ISSUE TITLE
fails generaten sql when searchValue has a multi letters within square brackets

### ISSUE THREAD

**Issue by chpo69** [borisdj/EFCore.BulkExtensions#100]
My code is something like this:
```
                        int nbUpdated = contextDb.Items.Where(tmp => tmp.NoTache == NoTache_AExpedier)
                                                 .BatchUpdate(new Item{ NoTache = NoTache_Expedie, DtModif = updatedDate });
```

the sql code produces :
```
UPDATE [t] SET [DtModif] = '23/11/2018 15:49:02', [NoTache] = '5'
FROM [Prod].[Item] AS [tmp]
WHERE [tmp].[NoTache] = 4
```

Problem it is  [t] name generated and not [tmp].

**Comment by borisdj** [comment]
Temp workaround is to write it with one letter:
```
int nbUpdated = contextDb.Items.Where(t => t.NoTache == NoTache_AExpedier)
                  .BatchUpdate(new Item{ NoTache = NoTache_Expedie, DtModif = updatedDate });
```

But I will see to fix it now.

**Comment by borisdj** [comment]
Fixed, try NuGet 2.3.0
